Luke7787/ImmaculateInventors
# Feature/fs folders

Summary
- Changed GET users endpoint to check for username and password
- Added GET request to get all folders for a given user
- Connected Backend + Mongo to the frontend
- Separated Item Boxes for folders and items
- Added dynamic routes using the folder ids
- Added auth context to sustain a user id

Testing
- `npm start` in root
- `npm run dev` in backend
- Sign in with credentials
   - Username: bossbaby  | Password: a1234567890!AA
- Navigate to http://localhost:3000/inventory  
   - Verify that 2 folders populate
   - Click edit button and add a new folder
   - Verify that the new folder is visible on the page
   - Click that new folder and make sure it's empty
   - Go back. click edit button and delete the folder you just created
   - Click each folder and verify that items populate
   - You can go back with the back button on the top left
   - In a folder, add a new item, verify that it populates, then delete it
- Note: Images don't work yet
